scripts/setupTypeScriptRollup.js

```diff
@@ -0,0 +1,135 @@
+// @ts-check
+
+/** This script modifies the project to support TS code in .svelte files like:
+  <script lang="ts">
+ 
+  As well as validating the code for CI.
+  */
+
+/**  To work on this script:
+  rm -rf test-template sapper-template && git clone sveltejs/sapper-template test-template && node scripts/setupTypeScript.js test-template
+*/
+
+const fs = require("fs")
+const path = require("path")
+const { argv } = require("process")
+
+const projectRoot = argv[2] || path.join(__dirname, "..")
+
+console.log("Adding typescript with rollup")
+
+if(fs.existsSync(path.join(projectRoot, "package.json"))) {
+
+  throw new Error("package.json already created. This script only works on plain new templates!")
+
+}
+
+// Add deps to pkg.json
+let pkgJSONPath = path.join(projectRoot, "package_template.json")
+const packageJSON = JSON.parse(fs.readFileSync(pkgJSONPath, "utf8"))
+packageJSON.devDependencies = Object.assign(packageJSON.devDependencies, {
+  "svelte-check": "^1.0.0",
+  "svelte-preprocess": "^4.0.0",
+  "@rollup/plugin-typescript": "^4.0.0",
```

Comment:
I'm recommending the svelte template switch TypeScript plugins. See https://github.com/sveltejs/template/pull/161

Reply:
FYI, I tried using `rollup-plugin-typescript2` and had trouble importing `.ts` files into `.svelte` files.  `@rollup/plugin-typescript@5.0.2` works just fine.